AngularVReact.md

```diff
@@ -1,5 +1,26 @@
 # Introduction
-Comparison points on Angular vs. React. Angular is full MVC while React is simply V.
+Before we get started, **I am in no way saying one framework is better than the other.** These are just some comparison points on Angular vs. React. In the end it all comes down to whatever you're looking for.
+
+Okay, read on.
+
+## Basic Concepts
+Angular is full MVC while React is simply V.
+
+**React:**
+
+- Virtual DOM: React is more of a UI render component than a full framework. Virtual DOM gives React 3 main advantages.
+
+1. Changes occur by comparing DOM and the virtual DOM. React only changes what is needed in the most optimal way.
+2. As we don't interact directly with the DOM, a browser is not needed to test React
+3. The virtual DOM has the ability to connect to other entities (Native and Electron).
+
+- State: Components in React have a state which represents component-related data. Updating state will allow the page to be reactive.
+
+**Angular:**
+
+- Watchers: Watchers (similar to event listeners) are attached to each component and each time a component is changed, watchers check if we should modify something else; and if needed, make appropriate modifications. This part in Angular 2 is way fast than its previous version so each time a component is changed, we don’t have to run any verifications on objects (depending on immutable elements).
+
+- Typescript: Angular2 requires Typescript. Compared to vanillaJS, Typescript offers better code organization, typing, and annotations so Typescript is better from a 'strictness' perspective but in regards to 'learning' you will often find yourself learning Typescript and Angular at the same time.
```

Comment:
I don't believe this is actually true. Angular 2 comes default with Typescript, but you can write in vanilla javascript if you so choose. 

see: [here](https://daveceddia.com/angular-2-in-plain-js/)